{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"func"},{"kind":"text","text":" "},{"kind":"identifier","text":"catchError"},{"kind":"text","text":"("},{"kind":"externalParam","text":"_"},{"kind":"text","text":" "},{"kind":"internalParam","text":"handler"},{"kind":"text","text":": "},{"kind":"attribute","text":"@escaping"},{"kind":"text","text":" (Swift"},{"kind":"text","text":"."},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":") "},{"kind":"keyword","text":"throws"},{"kind":"text","text":" -> "},{"kind":"typeIdentifier","identifier":"doc:\/\/com.apple.OPGGTestAPIKit\/documentation\/RxSwift\/Observable","preciseIdentifier":"s:7RxSwift10ObservableC","text":"Observable"},{"kind":"text","text":"<"},{"kind":"typeIdentifier","identifier":"doc:\/\/com.apple.OPGGTestAPIKit\/documentation\/RxSwift\/ObservableConvertibleType\/Element","preciseIdentifier":"s:7RxSwift25ObservableConvertibleTypeP7ElementQa","text":"Element"},{"kind":"text","text":">) -> "},{"kind":"typeIdentifier","identifier":"doc:\/\/com.apple.OPGGTestAPIKit\/documentation\/RxSwift\/Observable","preciseIdentifier":"s:7RxSwift10ObservableC","text":"Observable"},{"kind":"text","text":"<"},{"kind":"typeIdentifier","identifier":"doc:\/\/com.apple.OPGGTestAPIKit\/documentation\/RxSwift\/ObservableConvertibleType\/Element","preciseIdentifier":"s:7RxSwift25ObservableConvertibleTypeP7ElementQa","text":"Element"},{"kind":"text","text":">"}],"languages":["swift"],"platforms":["iOS"]}]},{"kind":"content","content":[{"anchor":"return-value","level":2,"type":"heading","text":"Return Value"},{"type":"paragraph","inlineContent":[{"type":"text","text":"An observable sequence containing the source sequence’s elements, followed by the elements produced by the handler’s resulting observable sequence in case an error occurred."}]}]},{"kind":"parameters","parameters":[{"name":"handler","content":[{"type":"paragraph","inlineContent":[{"type":"text","text":"Error handler function, producing another observable sequence."}]}]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/rxswift\/observabletype\/catcherror(_:)-swift.method"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/com.apple.OPGGTestAPIKit\/documentation\/RxSwift\/ObservableType\/catchError(_:)-swift.method","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Continues an observable sequence that is terminated by an error with the observable sequence produced by the handler."}],"kind":"symbol","metadata":{"modules":[{"name":"RxSwift"}],"role":"symbol","title":"catchError(_:)","roleHeading":"Instance Method","fragments":[{"kind":"keyword","text":"func"},{"kind":"text","text":" "},{"kind":"identifier","text":"catchError"},{"kind":"text","text":"((Swift"},{"kind":"text","text":"."},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":") "},{"kind":"keyword","text":"throws"},{"kind":"text","text":" -> "},{"kind":"typeIdentifier","text":"Observable","preciseIdentifier":"s:7RxSwift10ObservableC"},{"kind":"text","text":"<"},{"kind":"typeIdentifier","text":"Element","preciseIdentifier":"s:7RxSwift25ObservableConvertibleTypeP7ElementQa"},{"kind":"text","text":">) -> "},{"kind":"typeIdentifier","text":"Observable","preciseIdentifier":"s:7RxSwift10ObservableC"},{"kind":"text","text":"<"},{"kind":"typeIdentifier","text":"Element","preciseIdentifier":"s:7RxSwift25ObservableConvertibleTypeP7ElementQa"},{"kind":"text","text":">"}],"symbolKind":"method","externalID":"s:7RxSwift14ObservableTypePAAE10catchErroryAA0C0Cy7ElementQzGAIs0F0_pKcF","extendedModule":"RxSwift","platforms":[]},"hierarchy":{"paths":[["doc:\/\/com.apple.OPGGTestAPIKit\/documentation\/RxSwift","doc:\/\/com.apple.OPGGTestAPIKit\/documentation\/RxSwift\/ObservableType"]]},"references":{"doc://com.apple.OPGGTestAPIKit/documentation/RxSwift/ObservableConvertibleType/Element":{"role":"symbol","title":"Element","fragments":[{"kind":"keyword","text":"associatedtype"},{"kind":"text","text":" "},{"kind":"identifier","text":"Element"}],"abstract":[{"type":"text","text":"Type of elements in sequence."}],"identifier":"doc:\/\/com.apple.OPGGTestAPIKit\/documentation\/RxSwift\/ObservableConvertibleType\/Element","kind":"symbol","required":true,"type":"topic","url":"\/documentation\/rxswift\/observableconvertibletype\/element"},"doc://com.apple.OPGGTestAPIKit/documentation/RxSwift/ObservableType":{"role":"symbol","title":"ObservableType","fragments":[{"kind":"keyword","text":"protocol"},{"kind":"text","text":" "},{"kind":"identifier","text":"ObservableType"}],"abstract":[{"type":"text","text":"Represents a push style sequence."}],"identifier":"doc:\/\/com.apple.OPGGTestAPIKit\/documentation\/RxSwift\/ObservableType","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ObservableType"}],"url":"\/documentation\/rxswift\/observabletype"},"doc://com.apple.OPGGTestAPIKit/documentation/RxSwift":{"role":"collection","title":"RxSwift","abstract":[],"identifier":"doc:\/\/com.apple.OPGGTestAPIKit\/documentation\/RxSwift","kind":"symbol","type":"topic","url":"\/documentation\/rxswift"},"doc://com.apple.OPGGTestAPIKit/documentation/RxSwift/Observable":{"role":"symbol","title":"Observable","fragments":[{"kind":"keyword","text":"class"},{"kind":"text","text":" "},{"kind":"identifier","text":"Observable"}],"abstract":[{"type":"text","text":"A type-erased "},{"type":"codeVoice","code":"ObservableType"},{"type":"text","text":"."}],"identifier":"doc:\/\/com.apple.OPGGTestAPIKit\/documentation\/RxSwift\/Observable","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"Observable"}],"url":"\/documentation\/rxswift\/observable"},"doc://com.apple.OPGGTestAPIKit/documentation/RxSwift/ObservableType/catchError(_:)-swift.method":{"role":"symbol","title":"catchError(_:)","fragments":[{"kind":"keyword","text":"func"},{"kind":"text","text":" "},{"kind":"identifier","text":"catchError"},{"kind":"text","text":"((Swift"},{"kind":"text","text":"."},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":") "},{"kind":"keyword","text":"throws"},{"kind":"text","text":" -> "},{"kind":"typeIdentifier","text":"Observable","preciseIdentifier":"s:7RxSwift10ObservableC"},{"kind":"text","text":"<"},{"kind":"typeIdentifier","text":"Element","preciseIdentifier":"s:7RxSwift25ObservableConvertibleTypeP7ElementQa"},{"kind":"text","text":">) -> "},{"kind":"typeIdentifier","text":"Observable","preciseIdentifier":"s:7RxSwift10ObservableC"},{"kind":"text","text":"<"},{"kind":"typeIdentifier","text":"Element","preciseIdentifier":"s:7RxSwift25ObservableConvertibleTypeP7ElementQa"},{"kind":"text","text":">"}],"abstract":[{"type":"text","text":"Continues an observable sequence that is terminated by an error with the observable sequence produced by the handler."}],"identifier":"doc:\/\/com.apple.OPGGTestAPIKit\/documentation\/RxSwift\/ObservableType\/catchError(_:)-swift.method","kind":"symbol","type":"topic","deprecated":true,"url":"\/documentation\/rxswift\/observabletype\/catcherror(_:)-swift.method"}}}